asp_interactive_configuration/ooasp/tests/test_performance.py

```diff
@@ -4,31 +4,31 @@
 from ooasp.interactive import InteractiveConfigurator
 from ooasp.kb import OOASPKnowledgeBase
 
-def test_solve_multiple_elements():
-    """ create elements and solve under 30 seconds """
+# def test_solve_multiple_elements():
```

Comment:
why is (almost) everything commented-out in this file?

Reply:
I must have commented it out while running the tests multiple times. I will uncomment